README.md

```diff
@@ -1,4 +1,10 @@
-# Using Anaconda
+
+# IMPACT-T
```

Comment:
The change in the README looks unrelated to the PR, can you please open this separately if this is needed?

Reply:
I could revert, but is there anything against it?

Reply:
We just keep PRs concise and fix one thing at a time. That way, they get reviewed faster and one can `git cherry-pick` the changes around.